build/docker/postgres/init/users.sql

```diff
@@ -0,0 +1,15 @@
+-- ##############################################################################
+-- (C) Copyright IBM Corp. 2021
+--
+-- SPDX-License-Identifier: Apache-2.0
+-- ###############################################################################
+
+-- Create the users
+CREATE USER fhirserver WITH LOGIN encrypted password 'change-password';
+CREATE USER fhirbatch  WITH LOGIN encrypted password 'change-password';
+
+GRANT CONNECT ON DATABASE fhirdb TO fhirserver;
+GRANT CONNECT ON DATABASE fhirdb TO fhirbatch;
+
+GRANT USAGE ON SCHEMA fhirdata to fhirserver;
```

Comment:
without this line, I'm seeing the healthchecks fail due to 
```
Caused by: org.postgresql.util.PSQLException: ERROR: permission denied for schema fhirdata
  Position: 24
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2676)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2366)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:356)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:490)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:408)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:329)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:315)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:291)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:243)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.postgresql.ds.PGPooledConnection$StatementHandler.invoke(PGPooledConnection.java:441)
	at com.sun.proxy.$Proxy107.executeQuery(Unknown Source)
	at com.ibm.ws.rsadapter.jdbc.WSJdbcStatement.executeQuery(WSJdbcStatement.java:574)
	at com.ibm.fhir.database.utils.schema.GetSchemaVersion.run(GetSchemaVersion.java:45)
	... 59 more
```

should our schema tool be granting this USAGE?

Reply:
GetSchemaVersion.java:45 is a select: 
 https://github.com/LinuxForHealth/FHIR/blob/main/fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/schema/GetSchemaVersion.java#L43

Reply:
I saw that we added a "grantSchemaUsage" method for the createSchema action under https://github.com/LinuxForHealth/FHIR/issues/3417

I've now added a commit so that we will do the same grant if `--grant-to` is passed during an --update-schema or --update-schema-x action